### ISSUE TITLE
Suggestion ;)

### ISSUE THREAD

**Issue by Marshxan** [top-gg/issues#796]

**Comment by ExtremeDevelopments** [comment]
You didn't provide any content for your suggestion...

**Comment by Marshxan** [comment]
as a suggestion add more things to premium users because 2.99 just to block 10S ads is kind of useless so add more features like
1. custom channel unlocked in the discord
2. you unlock features on the website like promo for a server/bot for 24 hours
3. custom role in discord

**Comment by ExtremeDevelopments** [comment]
There are plans to add more stuff to premium, although details are not yet released. http://img.extreme-is.me/NNhCeCgWjrlVGjKN

**Comment by Marshxan** [comment]
is there a rough ETA :))

**Comment by ExtremeDevelopments** [comment]
No, there is no ETA at the moment. I am sure there will be some sort of announcement or update though once further plans are made! :)
